framework-docs/modules/ROOT/pages/core/aop/proxying.adoc

```diff
@@ -205,9 +204,12 @@ Java::
 ----
 	public class SimplePojo implements Pojo {
 
+	    @Autowired
+        private SimplePojo self;
```

Comment:
```suggestion
        private Pojo self;
```

Unless you are certain CGLIB proxies are being created, this would need to be `Pojo`.

Though, there's no need to update this PR. I'll address that after merging.